### ISSUE TITLE
[Accessibility] Microsite - missing Alt text on multiple images

### ISSUE THREAD

**Issue by emmaindal** [backstage/backstage#21882]
### 📜 Description

Missing on some pages (Docs); heading text repeated in alt on others (Plug-ins).

This helps those who use screen readers or are on slower connections. Alt text should be defined based on image type and the [w3C Alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).

### 👍 Expected behavior

Images to have alternate text. [Learn more about the alt attribute](https://dequeuniversity.com/rules/axe/4.8/image-alt).

### 👎 Actual Behavior with Screenshots

Example: 
<img width="590" alt="Screenshot 2023-12-15 at 13 12 23" src="https://github.com/backstage/backstage/assets/25153114/e60e7549-7b82-4516-b786-229b990de15a">

### 👟 Reproduction steps

1. Go to https://backstage.io/
2. [Run the Lighthouse browser extension ](https://chromewebstore.google.com/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?pli=1)

Alternatively:
1. Run the microsite locally
2. [Run the Lighthouse CLI locally when developing new features](https://backstage.io/docs/accessibility/#run-the-lighthouse-cli-locally-when-developing-new-features)

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by awanlin** [comment]
Hi @2div, I've assigned it to you, thanks 👍

**Comment by npiyush97** [comment]
@awanlin https://github.com/backstage/backstage/pull/21937#issue-2048597921 already a pull request.

**Comment by awanlin** [comment]
My apologies @npiyush97, I should have read more, you have clearly already submitted a PR 👍

**Comment by 2div** [comment]
> Hi @2div, I've assigned it to you, thanks 👍

Hi @awanlin Thank you for your reply.

I just that there is PR for this issue.
